docs/elements-vs-dev-portal.md

```diff
@@ -0,0 +1,122 @@
+# Elements vs Elements Dev Portal
```

Comment:
Have a bunch of thoughts on this one but no time to write them down atm - will try to tomorrow! If I forget, ping me.

Reply:
I think I just copied over the existing contents. What are your thoughts?

Reply:
Took a quick pass, lmk what you think. Still not great but overall this PR is much better than before, so it's a good merge!